packages/x-date-pickers/src/CalendarPicker/CalendarPicker.tsx

```diff
@@ -444,7 +477,6 @@ const CalendarPicker = React.forwardRef(function CalendarPicker<TDate>(
               reduceAnimations={reduceAnimations}
               selectedDays={[date]}
               onSelectedDaysChange={onSelectedDayChange}
-              loading={loading}
```

Comment:
What's the reason for removing `loading`?

Reply:
It's not used anywhere else in the component and is not coming from `...other` where all the defaultized props live.

Reply:
I was looking for `{...other}` at the end of the props 😅

Maybe could do the same with `renderLoading` such that they are both handled the same way

Reply:
Indeed, `renderLoading` is only used in `DayPicker` so the default value is useless here

Reply:
Done